config/bidderinfo.go

```diff
@@ -338,14 +338,16 @@ func validateAdapterEndpoint(endpoint string, bidderName string, errs []error) [
 	return errs
 }
 
-func validateInfo(info BidderInfo, bidderName string) error {
+func validateInfo(info BidderInfo, infos BidderInfos, bidderName string) error {
```

Comment:
Please differentiate the names `info` and `infos` more. The single character difference is too confusing.

Reply:
Yeah I agree with this

Reply:
updated in [1081cc7](https://github.com/prebid/prebid-server/pull/2996/commits/1081cc7870716a3bfb638a8d225726b8115a9a5e)